pkg/workflow/controller/handlers/workflowTrigger/handler.go

```diff
@@ -0,0 +1,71 @@
+package workflowTrigger
+
+import (
+	"github.com/caicloud/cyclone/pkg/apis/cyclone/v1alpha1"
+	"github.com/caicloud/cyclone/pkg/workflow/controller/handlers"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Selector is a selector of WorkflowTrigger, it defines the logic
+// to judge whether a WorkflowTrigger meet some conditions.
+type Selector func(wfr *v1alpha1.WorkflowTrigger) bool
```

Comment:
Please refer to latest develop branch code to remove `Selector` related code, it's not suitable here actually.